internal/rego/rego.go

```diff
@@ -0,0 +1,26 @@
+package rego
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+// File is a parsed Rego file
+type File struct {
+	FilePath       string
+	PackageName    string
+	ImportPackages []string
+	Contents       string
+	RulesActions   []string
+	Comments       []string
+}
+
+// Kind returns a Kubernetes Kind from the File's path
+func (f *File) Kind() string {
```

Comment:
Since I would expect this package to be a generic `rego` file, a `rego` in itself isn't going to have the concept of a Kind-- Kind is a part of another domain (Kubernetes/Gatekeeper).

I think it'd make more sense to remove this and have the CLI handle this piece.